src/dotnet/APIView/APIView/Languages/CodeFileBuilder.cs

```diff
@@ -319,9 +319,7 @@ private void BuildMember(CodeFileTokensBuilder builder, ISymbol member)
                 !member.IsAbstract &&
                 member.ContainingType.TypeKind != TypeKind.Interface)
             {
-                builder.Space();
-                builder.Punctuation(SyntaxKind.OpenBraceToken);
-                builder.Punctuation(SyntaxKind.CloseBraceToken);
+                builder.Punctuation(SyntaxKind.SemicolonToken);
```

Comment:
This method can be simplified, there are multiple branches printing semicolon and you don't need the complicated logic anymore.

Reply:
There are 2 branches that are printing the semicolon:
 - if you have a non-abstract method that is defined on a class
 - if you don't have a property

There are ways to optimize that code, but I am not sure if we want to do that. Unless we make the method print `;` in more situations, I don't know of a much cleaner way to write this.

Combining the statements looks weirder to me:
```csharp
if ((member.Kind == SymbolKind.Method && !member.IsAbstract && member.ContainingType.TypeKind != TypeKind.Interface) 
    || (member.Kind != SymbolKind.Property))
{
    builder.Punctuation(SyntaxKind.SemicolonToken);
}
```

Reply:
I think you can entirely remove this branch and get the desired behavior.